### DISCUSSION TITLE
想问一下如何能编译出中文

### DISCUSSION THREAD

**Discussion by yangandk** [xjtu-blacksmith/easymcm#19] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/91403015/212080694-061a8e3a-1751-48c4-853e-e6584c2f7203.png)
想请教一下怎么能编译出中文，现在只有英文能够显示。

**Comment by xjtu-blacksmith** [comment, marked as answer]
您好！按我个人往年经验，论文中没有用到中文的场合，不知近年来情况是否有变化。正因如此，当初改写模板时删除了中文支持相关的内容，以减少编译时间和其他可能出现的问题。近几年来也没有用户提出相应的需求。

如果一定需要使用中文，请在导言区手工引用 `xeCJK` 宏包，然后使用 `xelatex` 命令编译文档；或者，也可以直接将文档类 `article` 改成 `ctexart` 试试。我这里暂时没有测试条件，请尝试后及时反馈结果，谢谢！

**Reply by yangandk** [reply]
我们想先将文章写成中文的编译后看看页数，按照您的方法已经实现了，谢谢！
